en/components/chips.md

```diff
@@ -16,17 +16,17 @@ Use the Chip Component as a compact visual shown in an obround to display tags a
 
 The Chip comes in three density variants:
 
-- Comfortable - Suitable for intractive scenarios e.g. action triggers
+- Comfortable - Suitable for interactive scenarios e.g. action triggers
 - Cosy - Appropriate for short areas such as the recipient field of an email message composer
-- Compact - Best for data dense scenarios such the values for a cell in the Grid
+- Compact - Best for data-dense scenarios such as the values for a cell in the Grid
 
 <img class="responsive-img" src="../images/chip_comfy.png" srcset="../images/chip_comfy@2x.png 2x" />
 <img class="responsive-img" src="../images/chip_cosy.png" srcset="../images/chip_cosy@2x.png 2x" />
 <img class="responsive-img" src="../images/chip_compact.png" srcset="../images/chip_compact@2x.png 2x" />
 
 ## Chip State
 
-When the user interacts with the Chip, it goes through various states: **idle**, selected when the user has performed selective action, and focused while the user is navigating through the tab index. These flexibility enhancements afford a more dynamic interaction design that can seamlessly flow into high-fidelity prototyping.
+When the user interacts with the Chip, it goes through various states: **idle**, selected when the user has performed selective action, and **focus** while the user is navigating through the tab index. These flexibility enhancements afford a more dynamic interaction design that can seamlessly flow into high-fidelity prototyping.
```

Comment:
```suggestion
When the user interacts with the Chip, it may change from the default **idle** state to focus and vice versa. This affords a more dynamic interaction design that can seamlessly flow into high-fidelity prototyping.
```